docs/docs/0.7.1/documentation/dps-and-data-model.md

```diff
@@ -235,6 +253,50 @@ spec:
 
 On Kubernetes the [`Dataplane`](#dataplane-entity) entity is also automatically created for you, and because transparent proxying is being used to communicate between the service and the sidecar proxy, no code changes are required in your applications.
 
+Kuma Control Plane also takes care of HttpGet probes. By default, webhook will override probe with a virtual one. So for the given probe:
```

Comment:
why Kuma needs to take care of HttpGet probes? what is webhook?

Reply:
@lobkovilya can you clarify?